yt_dlp/extractor/rtvslo.py

```diff
@@ -0,0 +1,150 @@
+
+        formats = []
+        if media.get('addaptiveMedia', False):
+            formats = self._extract_wowza_formats(
+                traverse_obj(media, ('addaptiveMedia', 'hls_sec'), expected_type=str),
+                v_id, skip_protocols=['smil'])
+        for strm in ('http', 'https'):
+            for f in media.get('mediaFiles'):
+                if traverse_obj(f, ('streams', strm)):
+                    formats.append({
+                        'bitrate': f.get('bitrate'),
+                        'url': traverse_obj(f, ('streams', strm)),
+                        'filesize': f.get('filesize'),
+                        'width': f.get('width'),
+                        'height': f.get('height'),
+                        'ext': f.get('mediaType').lower(),
+                        'format_id': f'files_{strm}_{f.get("mediaType").lower()}_{f.get("bitrate")}'
+                    })
+
+        if media.get('addaptiveMedia_sl', False):
+            for f in self._extract_wowza_formats(
+                traverse_obj(media, ('addaptiveMedia_sl', 'hls_sec')), v_id, skip_protocols=['smil']
+            ):
+                f.update({'format_note': 'Sign language interpretation', 'preference': -3})
+                formats.append(f)
+        for strm in ('http', 'https'):
+            for f in media.get('mediaFiles_sl'):
+                if traverse_obj(f, ('streams', strm)):
+                    formats.append({
+                        'bitrate': f.get('bitrate'),
+                        'url': traverse_obj(f, ('streams', strm)),
+                        'filesize': f.get('filesize'),
+                        'width': f.get('width'),
+                        'height': f.get('height'),
+                        'ext': f.get('mediaType').lower(),
+                        'format_id': f'files-sl_{strm}_{f.get("mediaType").lower()}_{f.get("bitrate")}',
+                        'format_note': 'Sign language interpretation',
+                        'preference': -3
+                    })
+
+        self._sort_formats(formats)
+
+        if any('intermission.mp4' in x.get('url', '') for x in formats):
+            self.raise_geo_restricted(countries=self._GEO_COUNTRIES, metadata_available=True)
+        if any('dummy_720p.mp4' in x.get('manifest_url', '') for x in formats) and meta.get('stub', '') == 'error':
+            raise ExtractorError(f'{self.IE_NAME} said: Clip not available')
+
+        info = {
+            'thumbnails': thumbs,
+            'subtitles': subs,
+            'title': meta.get('title'),
+            'id': v_id,
+            'description': meta.get('description'),
+            'formats': formats,
+            'timestamp': date,
+            'release_timestamp': unified_timestamp(meta.get('recordingDate')),
+            'duration': meta.get('duration') or parse_duration(meta.get('length')),
+            'webpage_url': ''.join(traverse_obj(meta, ('canonical', ('domain', 'path')))),
```

Comment:
needed?

Reply:
GitHub highlights seem to be slightly broken here, I'm seeing different code sections highlighted between the email and the web interface.
Concerning `intermission.mp4` and `dummy_720p.mp4`: if I don't filter for them, an error video will be downloaded. If the user doesn't check the returned video, they will believe the video was downloaded successfully and be surprised once they actually view it (which may be after the video is unpublished).

Reply:
Oh my, now GitHub's finally working. 

Reassembling `webpage_url` like this is probably expendable - however, accessing a link to a video on `4d.rtvslo.si` currently returns a redirect; and judging by its main page, the portal may be sunset some time in the future. So, I assemble a link to the current portal (`365.rtvslo.si`) from the data I receive.